frontend/src/modules/Dashboard/Components/Dashboard.tsx

```diff
@@ -171,25 +178,24 @@ export const Dashboard = () => {
             numeric: true,
           })
         })
-      case 'no-check-in-email':
-        return courseInfo.filter(hasUnsentCheckIns)
-      case 'no-no-match-email':
-        return courseInfo.filter(hasUnsentNoMatch)
       default:
         return courseInfo
     }
   }
-
-  const handleChange = (event: SelectChangeEvent) => {
+  const handleSortedChange = (event: SelectChangeEvent) => {
     setSortedOrder(event.target.value as SortOrder)
   }
+  const handleFilterChange = (event: SelectChangeEvent) => {
+    setFilteredOption(event.target.value as FilterOption)
+  }
 
   const [selectedRoster, setSelectedRoster] = useState<string>('FA22')
 
-  const sortedCourses = sorted(
+  const filteredCourses = filtered(
     courses.filter((course) => course.roster === selectedRoster),
-    sortedOrder
+    filteredOption
   )
+  const sortedCourses = sorted(filteredCourses, sortedOrder)
```

Comment:
You can just combine these into two variables and name it appropriately so it's less confusing.